### ISSUE TITLE
Unit Collision/Pathing on 8+ mobs pack

### ISSUE THREAD

**Issue by kmkmkm0** [SunwellTracker/issues#2499]
Decription: Pack with more than 8 mobs do weird thing .

How it works:  You cant always place them properly for tanking . They move to your back or place themself in circle arround you .

How it should work: You should be able to stack them a little more . their spacing is weird and why do they run to player back .

Source (you should point out proofs of your report, please give us some source): https://plays.tv/video/5dcacc6d528956aedf/10-mob-weird-shit-happen

**Comment by Aridam** [comment]
It's called "Mob Fanning". Legit mechanic. Look it up.

**Comment by kmkmkm0** [comment]
Thanks for the quick answer .

So it s a feature; i could only find very fews other private with it . 
But what the point of it i dont get it .
Retail ain t like this , and all it do is greatly nerf melee aoe farm .
Enlighten me please . Thanks in advance.

**Comment by Aridam** [comment]
Basically, this: https://youtu.be/GDAldody2YY?t=190
It's a Retail-era video from 2009 (WotLK) that was made for a different reason, but it happens to be an excellent showcase of Mob Fanning. See for yourself.

Here is a good bugreport with plenty of information found in some subsequent comments there, straight off the actual TrinityCore bugtracker: https://github.com/TrinityCore/TrinityCore/issues/18512

And here is an astounding TankSpot guide (also from WotLK) with explanations of the various Mob mechanics (like movement prediction, seamless chasing _until_ you've stopped moving, stepping back from you if too close, etc.) during Combat, one of which is about the Mob Fanning mechanic.

**Comment by kmkmkm0** [comment]
Your  first video is what i m talking about , the guy is tanking 10 mobs just like i did in my video .
Dont you see the difference ?
He can manage to keep mobs infront in a nice 180° radius to have his defence up ; they dont run in his back for no reason or circle him .

**Comment by Aridam** [comment]
Yes, you're correct -- your snippet shows some discrepancies with how the Mobs fan out around you (some try to go behind you). Not sure what the cause is myself, but it could definitely use some polish.

I guess this can still be considered a valid bugreport, since despite answering your questions and clearing up the confusion, what you provided actually shows something that's not correct about it -- the videos and TC report I linked can serve as information on how it should work more accurately compared to what you got on film here.

**Comment by Aridam** [comment]
The TankSpot video guide is actually the best video evidence of how Mob Fanning -- and most if not all other Mob movement and positioning features -- should work.
